clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10850,6 +10850,8 @@ def err_sycl_restrict : Error<
   "|call a dllimport function"
   "|call a variadic function"
   "|call an undefined function without SYCL_EXTERNAL attribute"
+  "|use a const static variable that is neither zero-initialized "
+  "nor constant-initialized"
```

Comment:
Can we mention global variables as well?
```suggestion
  "|use a const static or global variable that is neither zero-initialized "
  "nor constant-initialized"
```

Reply:
Done.